CHANGELOG.md

```diff
@@ -1,29 +1,14 @@
 # Changelog
-All notable changes to this project 5.9.x per each release will be documented in this file.
+All notable changes to this project 5.10.x per each release will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [v5.9.0.3] - 2020-03-12
-
-### Changed
-- Align Docker Compose resources with WSO2 Software License (WSL).
-- Set Docker Compose resources to use WSO2 Private Docker Registry Docker images, by default.
-
-## [v5.9.0.2] - 2020-01-21
-
-### Fixed
-- Fix MySQL connector jar download URL in v5.9.0 
-
-## [v5.9.0.1] - 2019-10-11
+## [v5.10.0.1] - 2020-03-18
```

Comment:
@ThilinaManamgoda minor change. No need to fix the date as of now since, we are not releasing today.

Reply:
Let's keep this date for now and will send another PR when we do the release.